tensorbay/cli/log.py

```diff
@@ -24,19 +25,111 @@
 """
 
 
-def _implement_log(
+class _CommitNode:  # pylint: disable=too-few-public-methods
+    def __init__(self, commit: Commit):
+        self.commit: Commit = commit
+        self.child_commits: List[_CommitNode] = list()
+        self.available_child_num = 0
+        self.parent_commit: _CommitNode
```

Comment:
Are `child_nodes` and `parent_node` better?

Reply:
Yes sir～